hw3/task1/index.html

```diff
@@ -0,0 +1,305 @@
+                            <div class="performance__graph">
+
+
+
+                            </div>
+                            <div class="performance__x-axis">
+                                <ul>
+                                    <li>Sun</li>
+                                    <li>Mon</li>
+                                    <li>Tue</li>
+                                    <li>Wed</li>
+                                    <li>Thu</li>
+                                    <li>Fri</li>
+                                    <li>Sat</li>
+                                </ul>
+                            </div>
+                            <div class="performance__labels">
+                                <div class="performance__label-container">
+                                    <div class="performance__label-circle performance__label-circle--blue"></div>
+                                    <div class="performance__label-text">Instagram</div>
+                                </div>
+                                <div class="performance__label-container">
+                                    <div class="performance__label-circle performance__label-circle--green"></div>
+                                    <div class="performance__label-text">Facebook</div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+                <section class="apps">
+                    <div class="section-header">
+                        <h1 class="section-title">Installed apps
+                        </h1>
+                        <a href="">
+                            <div class="section-settings">
+                            </div>
+                        </a>
+                    </div>
+
+                    <div class="apps__table">
+
+                        <div class="apps__column-title"><span>Source</span></div>
+                        <div class="apps__column-title">Amount</div>
+                        <div class="apps__column-title">Status</div>
+                        <div class="apps__column-title">User ID</div>
+                        <div class="apps__column-title">Joined</div>
+                        <div class="apps__column-title">Group</div>
+                        <!--Zepplin begins-->
+                        <div class="apps__source">
+                            <div class="apps__logo-box">
+                                <img src="svg/logos/zepplin.svg" alt="">
+                            </div>
+                            Zepplin
+                        </div>
+                        <div class="apps__amount">686.00</div>
+                        <div class="apps__status">
+                            <div class="apps__status-container apps__status-container--active">Active</div>
+                        </div>
+                        <div class="apps__uid">114423</div>
```

Comment:
I don't believe that each column should have own style class. They look the same